Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 
 let package = Package(
     name: "tuist",
-    platforms: [.macOS(.v10_11)],
+    platforms: [.macOS(.v10_12)],
```

Comment:
This is a breaking change that we need to agree with. Since the latest version is 10.15, I think it's ok dropping support for 10.11. What do you think @tuist/core ?

Reply:
By upping from 10.11 to 10.12 we would loose support for Xcode 7